### DISCUSSION TITLE
Windows Bundle Misssing Version in EXE

### DISCUSSION THREAD

**Discussion by markdemich** [airsdk/Adobe-Runtime-Support#2502] [Q&A]
I created a windows bundle and noticed that my resultant exe does not have a version in it's metadata. When creating a INNOSETUP, I need a version.  It's automatic for INNOSETUP to pull it from an EXE, but now I'm forced to write some custom script to parse XML to pull it out of the application descriptor file.  It would be nice we could transfer that version number directly to EXE when creating it.

**Comment by ajwfrost** [comment]
Hi

I had noticed this as well, and wondered why it wasn't being set... I knew that there was some manipulation of the .exe happening, to set the icons etc. We've just looked at the code, and this is basically it -> there is code to set up the icon resource of the .exe file but nothing to set up the version info. It looks like this should be possible though, so we'll look to add this capability in.

The idea would be to take information from the app descriptor file:
- versionNumber
- name
- description
- copyright

and set these into the .exe file for the file "details" information: respectively,
- File version and Product version
- Product name
- File description
- Copyright

Note that the versionNumber in AIR descriptor files is restricted to 3 elements, so I think for "File version" we would just add a zero into the final part of that...

Let us know if that sounds suitable?

thanks

**Reply by itlancer** [reply]
Implemented with latest AIR 50.2.
Thanks!

**Comment by markdemich** [comment]
Sounds good to me.
